llvm/test/Transforms/DeadStoreElimination/stores-of-existing-values.ll

```diff
@@ -655,3 +655,81 @@ exit:
   call void @use(ptr %p) argmemonly
   ret void
 }
+
+define void @scalable_scalable_redundant_store(ptr %ptr) {
```

Comment:
Should we have at least one test here with a non-constant offset like in the tests in `offsetted-overlapping-stores.ll`?